README.md

```diff
@@ -0,0 +1,348 @@
+{x: 1, y: 3, z: 4}
+
+> concat(Identity('abc'), Identity('def'))
+Identity('abcdef')
+```
+
+<h4 name="empty"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L591">empty :: Monoid m => m -> m</a></code></h4>
+
+See [`empty`][empty].
+
+```javascript
+> empty('abc')
+''
+
+> empty([1, 2, 3])
+[]
+
+> empty({x: 1, y: 2})
+{}
+
+> empty(Identity([1, 2, 3]))
+Identity([])
+```
+
+<h4 name="map"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L612">map :: Functor f => (a -> b, f a) -> f b</a></code></h4>
+
+See [`map`][map].
+
+```javascript
+> map(x => x + 1, [1, 2, 3])
+[2, 3, 4]
+
+> map(x => x + 1, {x: 1, y: 2})
+{x: 2, y: 3}
+
+> map(x => x + 1, xs => xs.length)('abc')
+4
+
+> map(x => x + 1, Identity(42))
+Identity(43)
+
+> map(x => x + 1, Tuple('abc', 42))
+Tuple('abc', 43)
+```
+
+<h4 name="bimap"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L636">bimap :: Bifunctor f => (a -> b, c -> d, f a c) -> f b d</a></code></h4>
+
+See [`bimap`][bimap].
+
+```javascript
+> bimap(s => s.toUpperCase(), Math.sqrt, Tuple('abc', 100))
+Tuple('ABC', 10)
+```
+
+<h4 name="promap"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L648">promap :: Profunctor p => (a -> b, c -> d, p b c) -> p a d</a></code></h4>
+
+See [`promap`][promap].
+
+```javascript
+> promap(Math.abs, x => x + 1, Math.sqrt)(-100)
+11
+```
+
+<h4 name="ap"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L660">ap :: Apply f => (f (a -> b), f a) -> f b</a></code></h4>
+
+See [`ap`][ap].
+
+```javascript
+> ap([Math.sqrt, x => x * x], [1, 4, 9, 16, 25])
+[1, 2, 3, 4, 5, 1, 16, 81, 256, 625]
+
+> ap(s => n => s.slice(0, n), s => Math.ceil(s.length / 2))('Ramda')
+'Ram'
+
+> ap(Identity(x => x + 1), Identity(42))
+Identity(43)
+```
+
+<h4 name="of"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L678">of :: Applicative f => (f a, b) -> f b</a></code></h4>
+
+See [`of`][of].
+
+```javascript
+> of([], 42)
+[42]
+
+> of(Identity(0), 42)
+Identity(42)
+```
+
+<h4 name="chain"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L693">chain :: Chain m => ((a -> m b), m a) -> m b</a></code></h4>
+
+See [`chain`][chain].
+
+```javascript
+> chain(x => [x, x], [1, 2, 3])
+[1, 1, 2, 2, 3, 3]
+```
+
+<h4 name="chainRec"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L705">chainRec :: ChainRec m => (m x, (a -> c, b -> c, a) -> m c, a) -> m b</a></code></h4>
+
+See [`chainRec`][chainRec].
+
+```javascript
+> chainRec(
+.   [],
+.   (next, done, s) =>
+.     s.length === 2 ? [s + '!', s + '?'].map(done)
+.                    : [s + 'o', s + 'n'].map(next),
+.   ''
+. )
+['oo!', 'oo?', 'on!', 'on?', 'no!', 'no?', 'nn!', 'nn?']
+
+> chainRec(
+.   [],
```

Comment:
Thanks. __README.md__ is generated by `make release-*`, so I'll actually delete the file before merging this pull request. It's useful to be able to view the rendered readme when reviewing the pull request, though, so I'll keep the file for the time being.